docs/pages/includes/install-homebrew.mdx

```diff
@@ -0,0 +1,30 @@
+ <Notice type="danger">
+
+The Teleport package in Homebrew is not maintained by Teleport and we can't
+guarantee its reliability or security. We recommend the use of our [official
+Teleport packages](https://goteleport.com/teleport/download?os=mac).
+
+<ScopedBlock scope={["enterprise", "cloud"]}>
+
+The Homebrew version lacks the ability to maintain SAML/OIDC Authentication
+connector through the `tctl` binary. Administrators especially are recommended
+to install the official enterprise maintained versions.
```

Comment:
```suggestion
The Homebrew version of Teleport lacks the ability to maintain SAML/OIDC authentication
connectors through the `tctl` binary. We recommend installing the official Teleport Enterprise edition of `tctl`.
```
- Plural agreement and capitalization
- Since Teleport encourages admin-less RBAC and doesn't have a built-in admin role, I think we can ditch "Administrators" here and use the active voice with "We recommend".